optimum/intel/openvino/configuration.py

```diff
@@ -535,6 +552,12 @@ def __init__(
         model_type: str = "transformer",
         fast_bias_correction: bool = True,
         overflow_fix: str = "disable",
+        dataset: Optional[str] = None,
+        tokenizer: Optional[str] = None,
+        processor: Optional[str] = None,
+        trust_remote_code: bool = False,
+        matmul_sq_alpha: Optional[float] = None,
```

Comment:
Can you please rename `matmul_sq_alpha` to `smoothquant_alpha`

Reply:
Won't there be a confusion with `convolution_sq_alpha`?

Reply:
agreed offline to keep only one parameter.